.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,5 @@
 <!--
-Please run black to format your code.
-See https://networkx.org/documentation/latest/developer/contribute.html for details.
+Please use pre-commit to lint your code.
+For more details check out https://networkx.org/documentation/latest/developer/contribute.html#development-workflow
+step 2 and step 4.
```

Comment:
It looks like this should be step 1 and 4? (instead of 2 and 4).  Also, to make the lines shorter:
```suggestion
For more details check out step 1 and 4 of 
https://networkx.org/documentation/latest/developer/contribute.html#development-workflow
```